### DISCUSSION TITLE
global-link.min.js error

### DISCUSSION THREAD

**Discussion by FalconWu2017** [ofajs/discussion#22] [General]
html：   
~~~html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>开发工具</title>
    <!-- <script src="https://cdn.jsdelivr.net/gh/kirakiray/ofa.js@4.5.18/dist/ofa.min.js"></script> -->
    <!-- <script src="https://cdn.jsdelivr.net/gh/kirakiray/ofa.js/dist/ofa.min.js"></script> -->
    <script src="./js/ofa.min.js"></script>

    <script src="https://cdn.jsdelivr.net/gh/kirakiray/ofa.js@4.5.23/libs/global-link/dist/global-link.min.js"></script>
    <!-- <script src="./js/global-link.min.js"></script> -->
    
</head>
<body>
    <o-global-link href="./css/mycss.css"></o-global-link>
    <l-m src="idx-leader.html"></l-m>
    <idx-leader></idx-leader>
</body>
</html>
~~~   
随便点了几下：   
![image](https://github.com/user-attachments/assets/0abafebe-fed3-4e6b-bc4e-9612c980786b)

**Comment by FalconWu2017** [comment]
还有为什么会有两个github的请求？？  并且都是请求失败的。  
https://ofajs.github.io/ofa-errors/errors/zhft.json
https://ofajs.github.io/ofa-errors/errors/en.json

**Reply by kirakiray** [reply]
> 还有为什么会有两个github的请求？？ 并且都是请求失败的。 https://ofajs.github.io/ofa-errors/errors/zhft.json https://ofajs.github.io/ofa-errors/errors/en.json

😂 这两个是适配地区语言用的打印信息包，你当前的网络无法访问 github.io 导致的，可以忽略这个报错，不会影响内部运行结果。

**Reply by FalconWu2017** [reply]
能去掉不？或者让我下载到本地？  我这的开发环境网络连接github很不稳定，经常断。甚至很多客户根本就没有外网，所有东西都必须本地化。

**Reply by kirakiray** [reply]
> 能去掉不？或者让我下载到本地？ 我这的开发环境网络连接github很不稳定，经常断。甚至很多客户根本就没有外网，所有东西都必须本地化。

好，我等下加个更新。

**Reply by kirakiray** [reply]
已调整 ofa-errors，可以将报错信息本地化，请在dist 目录里选择对应文件。

https://github.com/ofajs/ofa-errors

**Reply by FalconWu2017** [reply]
这样放进去就行了么？用不用script引入？

![image](https://github.com/user-attachments/assets/51acc121-bf5f-4a2e-95d6-ea3beaebc6d9)

**Reply by kirakiray** [reply]
把这个下载下来，通过 `script` 标签引入后，报错信息就是离线而且是中文的了。

https://ofajs.github.io/ofa-errors/dist/ofa-error-zh.js

**Comment by kirakiray** [comment]
要将你本地的 ofa.js 更新到最新版本，这个 global link 组件是基于最新的 ofa.js 放开的 api 开发的。

https://github.com/kirakiray/ofa.js/blob/main/dist/ofa.min.js

**Reply by FalconWu2017** [reply]
没有更新版本号？？  而且还是报错   
![image](https://github.com/user-attachments/assets/ed53e2d4-0cf6-4c95-b111-c9e1e03f8187)

**Reply by kirakiray** [reply]
换上带版本号的试试：https://cdn.jsdelivr.net/gh/kirakiray/ofa.js@4.5.23/dist/ofa.min.js

**Reply by kirakiray** [reply]
您打开ofajs文件，看看版本号多少？

<img width="603" alt="image" src="https://github.com/user-attachments/assets/da640e8b-27ba-4102-89d3-da596a9cca88">

**Reply by FalconWu2017** [reply]
是4.5.23
![image](https://github.com/user-attachments/assets/1b0b8f2d-232b-44a6-ba5e-6dde4a57cf93)

**Reply by FalconWu2017** [reply]
![image](https://github.com/user-attachments/assets/62354199-1dcc-4ad5-a877-65b5bb2bad96)

**Reply by kirakiray** [reply]
> ![image](https://private-user-images.githubusercontent.com/34147999/379987382-62354199-1dcc-4ad5-a877-65b5bb2bad96.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk4MzYyNjUsIm5iZiI6MTcyOTgzNTk2NSwicGF0aCI6Ii8zNDE0Nzk5OS8zNzk5ODczODItNjIzNTQxOTktMWRjYy00YWQ1LWE4NzctNjViNWJiMmJhZDk2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI1VDA1NTkyNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU1YzM5YWM2MDBhZjBmMDYxMjU0MjE1MGY2NWRkNzdiMDhjMjUxOGNjZDI5ZjFmOTlhZDU0OTk4MmY2ZDJmMDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Wv3pMiSTCY92SjCSJCvX91lPYXMmv27TakmaZ1wJcyw)

感谢反馈，应该是没有模板的组件导致的问题，我改改

**Reply by kirakiray** [reply]
已经修复问题，将 global-link 组件更新一下：

https://cdn.jsdelivr.net/gh/kirakiray/ofa.js@4.5.24/libs/global-link/dist/global-link.min.js

**Comment by FalconWu2017** [comment]
升级到4.5.26.看起来好像没问题了。

**Reply by kirakiray** [reply]
👌 大于等于 4.5.24 都可以。
